content/about/about.fr.md

```diff
@@ -0,0 +1,68 @@
+---
+title: à propos de
```

Comment:
Don't forget, the title field should not be changed 🙈 (in all files)

Reply:
Normally on the last commit it this error is corrected